docs/templates.md

```diff
@@ -68,3 +68,17 @@ Sometimes the intention might be to lock the template on the UI so that the bloc
 
 - `all` — prevents all operations.
 - `insert` — prevents inserting new blocks, but allows moving existing ones.
+
+## Default Post Types
```

Comment:
Maybe "Existing Post Type"?

Reply:
I took the term from the Codex https://codex.wordpress.org/Post_Types might be good to match it.

Reply:
Right, but I was thinking of cases where a CPT is registered elsewhere by another person.

Reply:
Good point 👍